docs/configuration/monitoring.md

```diff
@@ -1,77 +1,78 @@
 # Monitoring
 
-Currently two monitoring solutions are supported:
+Currently, we support two monitoring solutions:
 
-* [**Crow**](#setting-up-monitoring-using-crow) - A simple, lightweight, and built-in monitoring solution.
-* [**Prometheus + Grafana + Loki**](#setting-up-monitoring-using-grafana-prometheus-and-loki) - A comprehensive third-party monitoring solution, recommended for more advanced usage.
+-   **[Crow](#setting-up-monitoring-using-crow):** A simple, lightweight, and built-in monitoring solution.
+-   **[Prometheus + Grafana + Loki](#setting-up-monitoring-using-grafana-prometheus-and-loki):** A comprehensive third-party monitoring solution, recommended for more advanced usage.
+
+The sections below provide details on how to set up each monitoring solution.
 
 ## Setting up monitoring using Crow
 
-Crow displays information about the projects pending analysis and the jobs currently running on Codacy.
+Crow displays information about the projects that are pending analysis and the jobs currently running on Codacy.
 
-The Crow tool is installed alongside Codacy after the helm chart is deployed to the cluster.
-It can be accessed through the `/monitoring` path of the url pointing to your Codacy
-installation, e.g. `http://codacy.company.org/monitoring`. You must set the
-`global.codacy.crow.url` value in your `values.yaml` file so that anchor links to your
-projects can be properly established inside `crow`.
+The Crow tool is installed alongside Codacy when the Helm chart is deployed to the cluster. By default, Crow is configured as follows:
 
-For example:
+-   Crow will be available on the `/monitoring` path of your Codacy installation URL, such as `http://codacy.company.org/monitoring`
+-   The default credentials to access Crow are:
 
-```yaml
-global:
-  codacy:
-    crow:
-      url: "http://codacy.example.com/monitoring"
-```
+    ```yaml
+    username: codacy
+    password: C0dacy123
+    ```
 
-### Configuring your credentials
+Follow the steps below to set up Crow and change the default configurations:
 
-We highly recommend that you define a custom password for the Crow installation. You can do this either through the `values.yaml` file or through a `--set` parameter during the Helm installation process. This parameter can be configured as follows:
+1.  Set the user name, password, and hostname to access the Crow PostgreSQL database [already created here](../requirements.md#preparing-postgresql-for-codacy):
 
-* Through a `--set` parameter:
+    ```yaml
+    crowdb:
+      postgresqlUsername: <--- codacy-db-username --->
+      postgresqlPassword: <--- codacy-db-password --->
+      host: <--- codacy-db-host --->
```

Comment:
@h314to is there any chance these Crow DB settings could be moved to the `values-production.yaml` file, at the end of the DB configurations for all the other components? Something like this:

```yaml
  [...]
  jobsdb:
    create: false
    postgresqlUsername: <--- codacy-db-username --->
    postgresqlDatabase: jobs # You need to create the DB manually
    postgresqlPassword: <--- codacy-db-password --->
    host: <--- codacy-db-host --->
    service:
      port: 5432
  crowdb:
    create: false
    postgresqlUsername: <--- codacy-db-username --->
    postgresqlDatabase: jobs # You need to create the DB manually
    postgresqlPassword: <--- codacy-db-password --->
    host: <--- codacy-db-host --->
    service:
      port: 5432
```

This would make it easier for the administrator to configure these settings in one go while editing the `values-production.yaml`, most probably by doing a search and replace to define these values all at once.